net-mail/notmuch/notmuch-0.30.ebuild

```diff
@@ -0,0 +1,212 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_OPTIONAL=1
+NEED_EMACS="24.1"
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit bash-completion-r1 distutils-r1 elisp-common eutils flag-o-matic \
+	pax-utils readme.gentoo-r1 toolchain-funcs
+
+DESCRIPTION="Thread-based e-mail indexer, supporting quick search and tagging"
+HOMEPAGE="https://notmuchmail.org/"
+SRC_URI="https://notmuchmail.org/releases/${P}.tar.xz
+	test? ( https://notmuchmail.org/releases/test-databases/database-v1.tar.xz )"
+
+LICENSE="GPL-3"
+# Sub-slot corresponds to major wersion of libnotmuch.so.X.Y. Bump of Y is
+# meant to be binary backward compatible.
+SLOT="0/5"
+KEYWORDS="~alpha ~amd64 ~ppc64 ~x86 ~x64-solaris"
+REQUIRED_USE="
+	nmbug? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	test? ( crypt emacs python valgrind )
+"
+IUSE="crypt doc emacs mutt nmbug python test valgrind"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+	doc? (
+		app-doc/doxygen
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/sphinx
+		sys-apps/texinfo
+	)
+"
+CDEPEND="
+	dev-libs/glib
+	dev-libs/gmime:3.0[crypt]
+	>=dev-libs/xapian-1.4.14:=
+	sys-libs/talloc
+	sys-libs/zlib
+	emacs? ( >=app-editors/emacs-${NEED_EMACS}:* )
+	python? ( ${PYTHON_DEPS} dev-python/cffi )
+"
+DEPEND="${CDEPEND}
+	test? (
+		>=app-editors/emacs-${NEED_EMACS}:*[libxml2]
+		app-misc/dtach
+		sys-devel/gdb
+		crypt? ( app-crypt/gnupg dev-libs/openssl )
+	)
+	valgrind? ( dev-util/valgrind )
+"
+RDEPEND="${CDEPEND}
+	crypt? ( app-crypt/gnupg )
+	mutt? (
+		dev-perl/File-Which
+		dev-perl/Mail-Box
+		dev-perl/MailTools
+		dev-perl/String-ShellQuote
+		dev-perl/Term-ReadLine-Gnu
+		virtual/perl-Digest-SHA
+		virtual/perl-File-Path
+		virtual/perl-Getopt-Long
+		virtual/perl-Pod-Parser
+	)
+	nmbug? ( dev-vcs/git )
+"
+
+DOCS=( AUTHORS NEWS README )
+DOC_CONTENTS="There are a few backward-incompatible changes between
+notmuch version 0.18 < x <= 0.18. Please consult the NEWS file (0.18
+section) before first use."
+MY_LD_LIBRARY_PATH="${WORKDIR}/${P}/lib"
```

Comment:
This is used only once, I think you're better off defining the path in `src_test`.